### PR TITLE
feat(presenter): data protection page

### LINKED ISSUE #69: 🚀 [Feature] Data Protection Page
## 🚀 Feature

Implement Data Protection Text as extra Page, link it accordingly (footer & all forms)

image

### PR DESCRIPTION
**Pull request by maeckes-infinitylabs**
## 🍰 Pullrequest

Adds data protection page and section
image

### Issues

- fixes #69 

### Todo

- [] fix data protection section test

### CODE REVIEW

**Comment by maeckes-infinitylabs** [comment]
@ulfgebhardt @Mogge 

i don't know why the test ist failing from DataProtection.test.ts

image

will have a look tomorrow with fresh eyes, otherwise will need help

File: presenter/src/pages/datenschutz.page.test.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import DataProtectionSection from '#components/sections/DataProtectionSection.vue'
+
+import DataPrivacyPage from './datenschutz.page.vue'
+
+describe('DataPrivacyPage', () => {
+  const wrapper = mount(VApp, {
+    slots: {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+      default: h(DataPrivacyPage),
```
Comment (ulfgebhardt) [inline presenter/src/pages/datenschutz.page.test.ts:14]:
```suggestion
      default: h(DataPrivacyPage as Component),
```

File: presenter/src/pages/datenschutz.page.test.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { mount } from '@vue/test-utils'
+import { describe, it, expect } from 'vitest'
+import { h } from 'vue'
```
Comment (ulfgebhardt) [inline presenter/src/pages/datenschutz.page.test.ts:3]:
```suggestion
import { Component, h } from 'vue'
```

File: presenter/src/components/sections/DataProtectionSection.stories.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import DataProtectionSection from './DataProtectionSection.vue'
+
+import type { Meta, StoryObj } from '@storybook/vue3'
+
+// More on how to set up stories at: https://storybook.js.org/docs/vue/writing-stories/introduction
+const meta = {
+  title: 'Sections/DataProtectionSection',
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  component: DataProtectionSection,
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/DataProtectionSection.stories.ts:9]:
```suggestion
  component: DataProtectionSection as SBComp,
```

must also be imported `SBComp`

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**